lib/ex_aws/sts/auth_cache/assume_role_web_identity_adapter.ex

```diff
@@ -0,0 +1,78 @@
+defmodule ExAws.STS.AuthCache.AssumeRoleWebIdentityAdapter do
+  @moduledoc """
+  Provides a custom Adapter which intercepts ExAWS configuration
```

Comment:
"... custom Adapter ~which~ _that_ intercepts ...", just to avoid repeating the "which".